e2e-tests/gef/cypress/integration/scenarios/Deal editors cannot submit to UKEF/maker_checker-should-not-submit-edited-deal-add-document.spec.js

```diff
@@ -0,0 +1,76 @@
+import relative from '../../relativeURL';
```

Comment:
can we change the parent folder name to 
`maker-checker-user-role`

Reply:
done